src/controllers/chefs.controller.ts

```diff
@@ -0,0 +1,59 @@
+import { Request, Response } from 'express';
+import * as chefHandler from '../handlers/chefs.handlers';
+
+export const addNewChef = async (req: Request, res: Response) => {
+    try {
+        const newChef = await chefHandler.addNewChef(req.body);
+        res.status(201).json(newChef);
+    } catch (error: any) {
+        res.status(500).json({ message: error.message });
+    }
+};
+
+export const getAllChefs = async (req: Request, res: Response) => {
+    try {
```

Comment:
You are also bringing the chefs that are deleted, please filter them here and in getChefById (in that function you will need to return 400 if the chef was deleted